tests/sklearn/test_sklearn_autolog.py

```diff
@@ -123,6 +117,11 @@ def fit_func_name(request):
     return request.param
 
 
+def _get_model_uri(name: str = MODEL_DIR) -> str:
+    last_active_run = mlflow.last_active_run()
+    return f"runs:/{last_active_run.info.run_id}/{name}"
```

Comment:
```suggestion
def _get_model_uri() -> str:
    return mlflow.last_logged_model().model_uri
```

On mlflow-3, this needs to be updated.